server/controllers/stock/reports/stock/expiration_report.js

```diff
@@ -76,12 +76,12 @@ async function stockExpirationReport(req, res, next) {
           totals.expired.quantity += lot.mvt_quantity;
           total += lot.value;
         } else {
-          lot.quantity_at_risk = (lot.S_RISK_QUANTITY * -1);
+          lot.quantity_at_risk = lot.mvt_quantity;
```

Comment:
The quantity at risk here should be the quantity that will expire at the current consumption rate.  The critical code is found here:

https://github.com/IMA-WorldHealth/bhima/blob/cdf85350d60f94f134f6e0ce7b0e0534ec88e8fb/server/controllers/stock/core.js#L787-L809

The value you are looking for is `S_RISK_QUANTITY`.  That should be the quantity of stock that will be lost at the current rate of consumption.  The amount of time that stock could have lasted, had it not expired, is in `S_RISK`.

Reply:
Done.